romeo_description/urdf/romeo_generated_urdf/romeo_eyes.xacro

```diff
@@ -56,8 +56,8 @@
 	<link name="REye">
 		<inertial>
 			<mass value="0.01"/>
-			<inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
-			<origin rpy="0 0 0" xyz="0 0 0"/>
+			<inertia ixx="1e-09" ixy="0" ixz="0" iyy="1e-09" iyz="0" izz="1e-09" />
+			<origin rpy="1.5708 -0 -3.14159" xyz="0 0 0"/>
```

Comment:
For the inertias we need any value I agree. It would be great to add those dummy inertiasautomatically.
You can merge it as is. It would be awesome if you could send some URgiven that the feature exist in the generator